lib/active_merchant/billing/gateways/adyen_checkout.rb

```diff
@@ -52,13 +52,16 @@ def refund(money, authorization, options={})
       end
 
       def store(credit_card, options={})
+        return details(options) if options[:three_ds_data]
```

Comment:
it seems we're using three different names for the same thing in this file: `three_ds`, `3ds`, `3DS2`. Can we do something about it?

Reply:
I see only `options[:three_ds_data]` and methods like `add_3ds_data` and `add_3ds_details`, there is also `allow3DS2` but this is an option from Adyen that we need to set to true in some cases (so we can't change the name of it).
I've changed this `add_3ds_data` into `add_three_ds_data` and `add_3ds_details` into `add_three_ds_details`

Reply:
one fewer, cool! 🎉